webapp-django/crashstats/signature/templates/signature/signature_summary.html

```diff
@@ -0,0 +1,171 @@
+<div class="panel">
+    <header class="title">
+        <h2>Operating System</h2>
+    </header>
+    <div class="content">
+        <table class="tablesorter data-table">
+            <thead>
+                <tr>
+                    <th class="header">Rank</th>
+                    <th class="header">Operating System</th>
+                    <th class="header">Count</th>
+                    <th class="header">%</th>
```

Comment:
I made it as a general comment before but I think it's better to do it inline. 
I think "Percentage" looks better than "%". The "%" notation is "lazy". 
If it was about money/currency I think the column should be "Amount" (or "Debt" or "Profit" or whatever) and not "$" (or "£").